VARIABLES.md

```diff
@@ -123,3 +123,4 @@ Name | Description | Type | Default | Required |
 | s3_lifecycle_policy_transition_period | Number of days for transition to a different storage class using lifecycle policy. | string | "30" | No |
 | s3_storage_class | Destination S3 storage class for transition in the lifecycle policy. For valid values for S3 Storage classes, reference: https://www.terraform.io/docs/providers/aws/r/s3_bucket.html#storage_class | string | "INTELLIGENT_TIERING" | No |
 | s3_object_expiration_days | Number of days after which objects in Apiary managed schema buckets expire. | number | null | No |
+
```

Comment:
You need to add a line here to document the new `tags` entry.

Reply:
Also add a valid `tags` line to the example invocation in the `apiary_managed_schemas` section.